packages/abstract/src/core/collision/observer.ts

```diff
@@ -90,7 +90,9 @@ export class CollisionObserver<
         continue;
       }
 
+      // Make sure effects will re-run when those properties change
       entry.shape;
+      dragOperation.position.current;
```

Comment:
I don't know if it has a performance penalty, or why we don't simply remove the `untracked` call of `detectCollision`, so whatever signal used in `detectCollision` changes, the collision is recomputed.